.settings/com.google.gdt.eclipse.core.template.prefs

```diff
@@ -0,0 +1,4 @@
+eclipse.preferences.version=1
+jarsExcludedFromWebInfLib=*/src/test/resources/lib/appengine/appengine-api-labs.jar|*/src/test/resources/lib/appengine/appengine-api-stubs.jar|*/src/test/resources/lib/appengine/appengine-api.jar|*/src/test/resources/lib/appengine/appengine-remote-api.jar|*/src/test/resources/lib/appengine/appengine-testing.jar|*/src/test/resources/lib/httpunit/httpunit.jar|*/src/test/resources/lib/javamail/mail.jar|*/src/test/resources/lib/selenium/selenium-server-standalone-2.46.0.jar
+warSrcDir=src/main/webapp
+warSrcDirIsOutput=true
```

Comment:
Oh wait, the reason I used template file is because if there's a change to the actual `.prefs` file it will still be tracked (even though `.settings` is under `.gitignore`, the `.prefs` file was already committed and will be tracked). If we overwrite the original `.prefs` file it'll be shown as diff.